aws/hosted_zones.go

```diff
@@ -6,22 +6,24 @@ import "fmt"
 // http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints
 // It currently cannot be generated from the API json.
 var hostedZoneIDsMap = map[string]string{
-	"us-east-1":      "Z3AQBSTGFYJSTF",
-	"us-east-2":      "Z2O1EMRO9K5GLX",
-	"us-west-2":      "Z3BJ6K6RIION7M",
-	"us-west-1":      "Z2F56UZL2M1ACD",
-	"eu-west-1":      "Z1BKCTXD74EZPE",
-	"eu-west-2":      "Z3GKZC51ZF0DB4",
-	"eu-west-3":      "Z3R1K369G5AVDG",
-	"eu-central-1":   "Z21DNDUVLTQW6Q",
+	"ap-northeast-1": "Z2M4EHUR26P7ZW",
+	"ap-northeast-2": "Z3W03O7B5YMIYP",
+	"ap-northeast-3": "Z2YQB5RD63NC85",
 	"ap-south-1":     "Z11RGJOFQNVJUP",
 	"ap-southeast-1": "Z3O0J2DXBE1FTB",
 	"ap-southeast-2": "Z1WCIGYICN2BYD",
-	"ap-northeast-1": "Z2M4EHUR26P7ZW",
-	"ap-northeast-2": "Z3W03O7B5YMIYP",
 	"ca-central-1":   "Z1QDHH18159H29",
+	"eu-central-1":   "Z21DNDUVLTQW6Q",
+	"eu-north-1":     "Z3BAZG2TWCNX0D",
+	"eu-west-1":      "Z1BKCTXD74EZPE",
+	"eu-west-2":      "Z3GKZC51ZF0DB4",
+	"eu-west-3":      "Z3R1K369G5AVDG",
 	"sa-east-1":      "Z7KQH4QJS55SO",
+	"us-east-1":      "Z3AQBSTGFYJSTF",
+	"us-east-2":      "Z2O1EMRO9K5GLX",
 	"us-gov-west-1":  "Z31GFT0UA1I2HV",
```

Comment:
While we're making adjustments to this pull request, we can add `us-gov-east-1`, which has the same Route53 Hosted Zone ID as `us-gov-west-1` according to the [GovCloud (US-East) Endpoints documentation](https://docs.aws.amazon.com/govcloud-us/latest/ug-east/using-govcloud-endpoints.html)

```suggestion
	"us-gov-east-1":  "Z31GFT0UA1I2HV",
	"us-gov-west-1":  "Z31GFT0UA1I2HV",
```